downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/FileDownloadCallback.kt

```diff
@@ -37,15 +37,58 @@ import timber.log.Timber
 import java.io.File
 import javax.inject.Inject
 
-@ContributesBinding(AppScope::class)
+interface DownloadCallback {
```

Comment:
None of the methods below are currently used from the public API and probably they'll never will. So I have moved then to be private to the downloads implementation